server/item/enchantment/mending.go

```diff
@@ -0,0 +1,25 @@
+package enchantment
+
+import (
+	"github.com/df-mc/dragonfly/server/item"
+)
+
+// Mending is an enchantment that repairs the item when experience orbs are collected.
+type Mending struct{}
```

Comment:
Make sure to implement the Rarity function, introduced with anvils.

Reply:
Along with the `CompatibleWithEnchantment` function, and the rename from `CompatibleWith` to `CompatibleWithItem`.